### PR TITLE
tests: remove sanityCheckDeploymentsExist() lambda

### PR DESCRIPTION
**Pull request by codingben**
**What this PR does / why we need it**:

Remove uneseccary lambda function that contains
hard-coded parameter.

**Release note**:

```release-note
None
```

### CODE REVIEW

**Review by enp0s3 — COMMENTED** [review]
/approve

@codingben Thank you!

File: tests/operator/operator.go [hunk 1]
```diff
@@ -3442,6 +3424,18 @@ func nodeSelectorExistInDeployment(virtClient kubecli.KubevirtClient, deployment
 	return true
 }
 
+func sanityCheckDeploymentsExist(virtClient kubecli.KubevirtClient, namespace string) {
+	Eventually(func() error {
```
Comment (dankenigsberg) [inline tests/operator/operator.go:3428]: there is no need to expose these arguments as `kubevirt.Client()` returns a singleton, and the `namespace` is specific and constant to this test.

Comment (codingben) [reply tests/operator/operator.go:3428]: Right, I'll improve it.

**Review by dankenigsberg — APPROVED** [review]

**Review by enp0s3 — APPROVED** [review]

**Pull request merged**